package.json

```diff
@@ -36,6 +36,7 @@
     "react-motion": "^0.5.0",
     "react-redux": "5.0.6",
     "react-router-dom": "4.2.2",
+    "react-svg": "^2.1.21",
```

Comment:
I know I'm not reviewing yet, but this can go. Check out ICT or my branch to see SVG icon usage.